required/latex-lab/latex-lab-graphic.dtx

```diff
@@ -137,37 +384,211 @@
+%  but redefined here to switch to the illustrative mode and to add its value
+%  as alternative text. 
+%  \item[\texttt{actualtext}] This switches to the actualtext mode.
+%  This is useful for small graphics that represent single chars or a short word like a 
+%  logo. If \texttt{actualtext} is used, the graphics is not enclosed in \texttt{Figure}
+%  structure but in a \texttt{Span} structure and no \texttt{/BBox} attribute is added.%
+%  \footnote{This in accordance with (the draft of) PDF/UA-2 but violates perhaps PDF/UA-1.}
+%  \item[\texttt{artifact}] This tags the graphic as an artifact.
+%  \item[\texttt{tagging-setup}] This key takes as argument a key-list,
+%  which can contain the following keys:
 %   \begin{description}
-%    \item[\texttt{artifact}] When used the graphic will be tagged as artifact. This doesn't
-%     require a \texttt{BBox} and so works also in some of the not yet supported cases described
-%     above.
-%   \item[\texttt{false}] When used tagging will be stopped completely. It is then the 
+%    \item[\texttt{alt=\meta{text}}] This a second way to tag the graphic as figure with
+%    alternative text. \texttt{alt} without value will tag as figure but not change
+%    the text variable, \texttt{alt=} will empty the text variable (and typically trigger
+%    a warning).
+%    \item[\texttt{actualtext=\meta{text}}] This a second way to tag the graphic as a
+%    symbol with actualtext.  
+%    \item[\texttt{artifact}] This a second way to tag the graphic as artifact. 
+%    \item[\texttt{text}] This switches to text mode.
+%    \item[\texttt{off}] When used tagging will be stopped completely. It is then the 
 %   responsibility of the surrounding code to add appropriate tagging commands.
-%   \item[\meta{name}] Other values will be used as tag names in the structure. If the tag is not
-%   known as a structure tag you will get an warning from tagpdf. The default name is currently
-%   \texttt{Figure}
+%   \item[\texttt{tag=\meta{name}}] This switches to the illustrative mode but uses
+%   \meta{name} as tag name in the structure instead of the default \texttt{Figure}. 
+%   This can for example be used to tag an image of a formula with \texttt{Formula}.
 %   \end{description}   
-%  \item[\texttt{actualtext}] This allows to add an /ActualText to the structure. 
-%  This is useful for small graphics that represent single chars or a short word like a 
-%  logo. If \texttt{actualtext} is used, the graphics is not enclosed in \texttt{Figure}
-%  structure but in a \texttt{Span} structure and no \texttt{/BBox} attribute is added.
-%  This in accordance with (the draft of) PDF/UA-2 but violates perhaps PDF/UA-1.
 %  \item[\texttt{correct-BBox}] If the calculated \texttt{/BBox} values are wrong
-%  they can be correct with this key. It expects four dimensions that are added to the 
+%  they can be corrected with this key. It expects four dimensions that are added to the 
 %  \texttt{/BBox} values. 
```

Comment:
is that key really useful in the user interface? I mean nearly nobody would be able to figure out the right numbers.

Okay, I see there is also this "debug" interface which would help with that , so certainly better than nothing.